ibis/expr/tests/test_sql_builtins.py

```diff
@@ -82,15 +82,15 @@ def test_zeroifnull(functional_alltypes):
     iresult = functional_alltypes.int_col.zeroifnull()
 
     assert type(dresult.op()) == ops.ZeroIfNull
-    assert type(dresult) == ir.DoubleColumn
+    assert type(dresult) == ir.FloatingColumn
 
     # Impala upconverts all ints to bigint. Hmm.
     assert type(iresult) == type(iresult)
 
 
 def test_fillna(functional_alltypes):
     result = functional_alltypes.double_col.fillna(5)
-    assert isinstance(result, ir.DoubleColumn)
+    assert isinstance(result, ir.FloatingColumn)
```

Comment:
Because I've also deleted `FloatScalar`, `FloatColumn`, `DoubleScalar`, `DoubleColumn` in favor of `FloatingScalar` and `FloatingColumn`. `*Scalar` and `*Column` are only required to be able to bind methods on them in `api.py`. All of their properties are defined by the `datatype` they rely on.

Reply:
I see, so you haven't lost any information since all of it is contained by the `datatype`. Sounds good.